contracts/interfaces/IMatchingMarket.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.4.16;
+
+import "./DSToken.sol";
+
+interface IMatchingMarket {
```

Comment:
I think this should be an ISimpleMarket. Comparing [simple_market.sol](https://github.com/makerdao/maker-otc/blob/master/src/simple_market.sol) and [matching_market.sol ](https://github.com/makerdao/maker-otc/blob/master/src/matching_market.sol) you can see that the Matching Market doesn't have a getOffer() function.